### PR TITLE
Added mode for INI file

### PR DESCRIPTION
**Pull request by hirokiky**

### CODE REVIEW

File: kaa/filetype/ini/__init__.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from kaa.filetype import filetypedef
+
+
+class FileTypeInfo(filetypedef.FileTypeInfo):
+    FILE_EXT = {'.ini', '.conf'}
```
Comment (atsuoishimoto) [inline kaa/filetype/ini/__init__.py]: Can '.conf' files be able to considered ini files?  'http.conf' came into my mind at first.

File: kaa/filetype/ini/inimode.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+ini_keys = {
+    (ctrl, 't'): 'toc.showlist',
+}
+
+
+class IniMode(defaultmode.DefaultMode):
+    MODENAME = 'Ini'
+
+    RE_SECTION = doc_re.compile(r'\[(?P<SECTION>.+)\]')
```
Comment (atsuoishimoto) [inline kaa/filetype/ini/inimode.py]: Section should start at beginng of line.

**Comment by hirokiky** [comment]
@atsuoishimoto Fixed ploblems :accept:

**Comment by atsuoishimoto** [comment]
Merged. Thanks!

**Pull request merged**